<launch>
    <node name="hsv_filter_node" pkg="vision" type="hsv_filter_node" output="screen">
        <param name="h_low" value="0" type="int" />
        <param name="s_low" value="132" type="int" />
        <param name="v_low" value="15" type="int" />
        <param name="h_high" value="112" type="int" />
        <param name="s_high" value="215" type="int" />
        <param name="v_high" value= "106" type="int" />
        <remap from="/camera/image_raw" to="uwsim/camera2"/>
    </node>
</launch>
